src/meta-plane/inheritance_tree.py

```diff
@@ -0,0 +1,67 @@
+# Dunders __base__, __bases__, __subclasses__
+
+
+from __future__ import annotations
+
+
+class Carbon:
+    def get_atom(self) -> str:
+        return "C"
+
+
+class Hydrogen:
+    def get_atom(self) -> str:
+        return "H"
+
+
+class Oxygen:
+    def get_atom(self) -> str:
+        return "O"
+
+
+class Ozone(Oxygen):
+    ...
+
+
+class Water(Oxygen, Hydrogen):
+    ...
+
+
+class CarbonDioxide(Carbon, Oxygen):
+    ...
+
+
+class Glucose(CarbonDioxide, Water):
+    ...
+
+
+# Inheritance tree
+#
+#          Glucose
+#         /        \
+# CarbonDioxide     Water
+#       |     Ozone   |
+#       /\    |      /\
+#      /  \   |     /  \
+#     /    \  |    /    \
+# Carbon     Oxygen   Hydrogen
+
+
+def main():
+    print(f"Glucose derives from {Glucose.__bases__ = }")
+    print(f"CarbonDioxide derives from {CarbonDioxide.__bases__ = }")
+    print(f"Ozone derives from {Ozone.__base__ = }")
+
+    # __base__ is the first element of __bases__
+    # and the first in the inheritance list
+    print(f"Water first derives from {Water.__base__ = }")
+
+    # __base__ also determines which method is called
+    # if two methods have the same nade
```

Comment:
```suggestion
    # The MRO (method resolution order) is decided by checking the
    # `__bases__` array from left to right.
```
Not sure if line length is good

Reply:
Although, this is determined by `__mro__` rather than `__bases__`. I'm going to change the comment to something else still